src/action/channel.js

```diff
@@ -100,7 +101,6 @@ class ChannelAction {
       const { channels } = await this._grpc.sendCommand('listChannels');
       this._store.channels = channels.map(channel => ({
         remotePubkey: channel.remote_pubkey,
-        id: channel.chan_id,
```

Comment:
Should we remove this? Might be helpful to have the attribute internally even if we don't display it in the UI currently.